common.props

```diff
@@ -3,7 +3,7 @@
     <ContinuousIntegrationBuild Condition="'$(TF_BUILD)' == 'true'">true</ContinuousIntegrationBuild>
 
     <MajorProductVersion>4</MajorProductVersion>
-    <MinorProductVersion>6</MinorProductVersion>
+    <MinorProductVersion>7</MinorProductVersion>
```

Comment:
We don't need to rev versions for updates to test projects

Reply:
ohh yeah. I missed that

Reply:
corrected it